docker-image/Dockerfile.s390x

```diff
@@ -1,26 +1,36 @@
 ARG QEMU_IMAGE=calico/go-build:latest
 FROM ${QEMU_IMAGE} as qemu
 
-FROM s390x/alpine:3.8
+FROM s390x/debian:9.8-slim as base
 MAINTAINER LoZ Open Source Ecosystem (https://www.ibm.com/developerworks/community/groups/community/lozopensource)
 
 # Enable non-native builds of this image on an amd64 hosts.
-# This must be the first RUN command in this file!
-# we only need this for the intermediate "base" image, so we can run all the apk and other commands
+# This must before any RUN command in this image!
+# we only need this for the intermediate "base" image, so we can run all the apt-get and other commands
+# and this is only because of using older kernels
 # when running on a kernel >= 4.8, this will become less relevant
 COPY --from=qemu /usr/bin/qemu-s390x-static /usr/bin/
 
 # Since our binary isn't designed to run as PID 1, run it via the tini init daemon.
-RUN apk add --update tini
-ENTRYPOINT ["/sbin/tini", "--"]
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
```

Comment:
This likely won't work for other architectures since we're downloading a compiled binary that I assume is built for amd64.

Do we need tini at all?

Reply:
that's a question for @fasaxc 

I'd also like to understand the background story.

Reply:
> This likely won't work for other architectures since we're downloading a compiled binary that I assume is built for amd64.

Good point, that's my mistake. 

Tini provides statically built versions for each arch: 
https://github.com/krallin/tini/releases

I just need to update the URL to the appropriate version for each. 

> Do we need tini at all?

That's a bigger question! Do we have a clear way to rule out that Typha creating subprocesses?

Reply:
I decided to leave `tini` in place. Each Dockerfile has the correct `tini` binary for its arch.